crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -296,8 +297,12 @@ def proposal?(file)
         raw_data.key?("attributes") && raw_data.key?("deployment")
       end
 
-      def migrate_database(time)
-        Crowbar::Migrate.migrate!
+      def migrate_database(time, migration_level = nil)
+        if migration_level
+          Crowbar::Migrate.migrate_to(migration_level)
```

Comment:
> If we add a db migration over the lifetime of the product, it could happen if the user does a backup with updates, and restores without updates.

that's the point that I don't understand, why would he restore without the updates? wouldn't that mean that the updates were rolled back?